### DISCUSSION TITLE
[Feature] wallpaper/lock screen image cycling

### DISCUSSION THREAD

**Discussion by wsidl** [immich-app/immich#10097] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Android and iOS devices have an ability to have apps cycle images on the device's wallpaper and/or lock screen. 

This feature is asking for an ability to do the same with Immich by selecting an album (or smart album per #1673 ) and use that as the source of images to load 

## Use Case
I might have an album of my family, pets, exciting events, etc, and would like to have those appear on my wallpaper.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by warreth** [comment]
Any work on this? I would like to help!

**Reply by warreth** [reply]
@alextran1502 If i start working on this feature, will it get merged?
